test/test.js

```diff
@@ -3,6 +3,7 @@ var num2fraction = require('../')
 var π = Math.PI
 
 test('Must be equal', function(t) {
+  t.equal(num2fraction(1/0), '0')
```

Comment:
Sorry for the late reply, I think it makes more sense to return to `Infinity` here.

Reply:
Changes done